src/main/groovy/com/marklogic/gradle/task/MarkLogicTask.groovy

```diff
@@ -44,10 +44,14 @@ class MarkLogicTask extends DefaultTask {
 		project.hasProperty("mlAdminPassword") ? project.property("mlAdminPassword") : project.property("mlPassword")
 	}
 
+	String getRoxyHome(){
```

Comment:
Just define this in your task class - I don't think it will be reused enough to warrant its inclusion in MarkLogicTask, which is intended to have commonly reused methods.